src/main/java/Duke.java

```diff
@@ -1,10 +1,67 @@
+import java.util.Scanner;
+import java.util.*;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String line;
```

Comment:
'line" is quite a vague name, maybe you can give a more unique or clearer name to better understand what this variable is used for in the program